### PR TITLE
Update getting started

### PR DESCRIPTION
**Pull request by aakankshaduggal**
In this PR, we updated the Getting Started section of the project documentation and create documentation from the standpoint of different personas.

We also remove unnecessary template documents from the repo and update the readme to include the project name/shorthand AI4CI.

## Related Issues and Dependencies

https://github.com/aicoe-aiops/ocp-ci-analysis/issues/340
https://github.com/aicoe-aiops/ocp-ci-analysis/issues/339

## This introduces a breaking change

- [ ] Yes
- [x] No

### CODE REVIEW

File: docs/data-science-get-started.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+## Leverage Open Data Sources
+
+One of the goals of the AI4CI project is to build an open AIOps community centered around open datasets. As a part of this project, we interact with various open source CI data sources and make them available for others to leverage. A detailed overview of our efforts with the various data sources we are working with, along with data collection scripts, and exploratory analyses is available [here](content.md#data-engineering-metrics-and-kpis-for-ci).
+
+## Try out the Notebooks
+
+There are interactive and reproducible notebooks for this entire [project](https://github.com/aicoe-aiops/ocp-ci-analysis) available for anyone to start using on the public [JupyterHub](https://jupyterhub-opf-jupyterhub.apps.zero.massopen.cloud/hub/login) instance on the [Massachusetts Open Cloud](https://massopen.cloud/) (MOC) right now!
+
+1. To get started, access [JupyterHub](https://jupyterhub-opf-jupyterhub.apps.zero.massopen.cloud/), select log in with `moc-sso` and sign in using your Google Account.
```
Comment (isabelizimm) [inline docs/data-science-get-started.md:13]: this link will need to be updated with smaug

File: docs/data-science-get-started.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+2. After signing in, on the spawner page, please select the `ocp-ci-analysis:latest` image in the JupyterHub Notebook Image section from the dropdown and select a `Medium` container size and hit `Start` to start your server.
+3. Once your server has spawned, you should see a directory titled `ocp-ci-analysis-<current-timestamp>`. Browse through, run the various notebooks and start exploring this project.
+4. To interact with the S3 bucket and access the stored datasets, make sure you have a `.env` file at the root of your repo. Check [.env-example](../.env-example) for an example `.env` file and open an [issue](https://github.com/aicoe-aiops/ocp-ci-analysis/issues) for access credentials.
+
+You can find more information on the various notebooks and their purpose [here](content.md).
+
+If you need more help navigating the Operate First environment, we have a few [short videos](https://www.youtube.com/playlist?list=PL8VBRDTElCWpneB4dBu4u1kHElZVWfAwW) to help you get started.
+Interact with Dashboards
```
Comment (isabelizimm) [inline docs/data-science-get-started.md:21]: should this be a new heading?

File: docs/data-science-get-started.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+As a part of AI4CI, we collect the relevant metrics and key performance indicators (KPIs) and visualize them using dashboards. You can view and interact with the publicly available dashboard [here](https://superset.apps.devconfus2021.aws.operate-first.cloud/superset/dashboard/ocp-ci-kpi-dashboard/).
+
+## Interact with Model Endpoints
+
+**Github Time to Merge Model**: We have an interactive endpoint available for a model which can predict the time taken to merge a PR and classifies it into one of a few predefined time ranges. To interact with the model, check out this [Model Inference Notebook](../notebooks/time-to-merge-prediction/model_inference.ipynb)
```
Comment (isabelizimm) [inline docs/data-science-get-started.md:26]: can we add in failure classifier or other endpoints?

File: docs/data-science-get-started.md (hunk 1 above)
Comment (oindrillac) [reply docs/data-science-get-started.md:13]: Yes, correct! :) we will also need to update model endpoints, JH URL, Kubeflow pipeline url etc

File: docs/data-science-get-started.md (hunk 2 above)
Comment (oindrillac) [reply docs/data-science-get-started.md:21]: yes, we should correct this.

File: docs/data-science-get-started.md (hunk 3 above)
Comment (oindrillac) [reply docs/data-science-get-started.md:26]: Thanks, we will go ahead and add those. Has the [failure type classifier model](https://github.com/aicoe-aiops/ocp-ci-analysis/pull/377) been deployed and is it ready to be shared? 
We will add the build log classifier model to this list.

Comment (isabelizimm) [reply docs/data-science-get-started.md:26]: I was thinking [build log](https://github.com/aicoe-aiops/ocp-ci-analysis/pull/371). I can add a PR to update this when failure type is merged :)

**Review by chauhankaranraj — COMMENTED** [review]
Looks great folks! :smile:  /lgtm

File: docs/kubernetes-contributor-get-started.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
+# Engage as a Kubernetes Contributor
+
+If you are a Kubernetes contributor and would like to engage with the [AI4CI](../README.md) project, here are a few ways to get started and we compile a list of ways you can engage with this project.
```
Comment (MichaelClifford) [inline docs/kubernetes-contributor-get-started.md:3]:
```suggestion
If you are a Kubernetes contributor and would like to engage with the AI4CI project, here are a few ways to get started.
```

File: docs/kubernetes-contributor-get-started.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+## Kubernetes CI/CD Data Sources
+
+### Analyzing TestGrid
+
+In order to quantify the current state of the CI workflow and identify any gaps within the CI process better, we calculate certain key performance indicator metrics related to the test runs using the data obtained from [TestGrid](https://testgrid.k8s.io/).
```
Comment (MichaelClifford) [inline docs/kubernetes-contributor-get-started.md:9]:
```suggestion
In order to quantify the current state of the CI workflow and identify any gaps within the CI process, we calculate certain key performance indicators and metrics related to the test runs using the data obtained from [TestGrid](https://testgrid.k8s.io/).
```

File: docs/kubernetes-contributor-get-started.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+
+Here is a short [demo](https://youtu.be/4Mlg2qijEgQ) on how we extract data from the testgrid UI and calculate metrics from it and display them on dashboards.
```
Comment (MichaelClifford) [inline docs/kubernetes-contributor-get-started.md:11]:
```suggestion
Here is a short [demo](https://youtu.be/4Mlg2qijEgQ) on how we extract data from the testgrid UI, calculate metrics from it and display them on dashboards.
```

File: docs/kubernetes-contributor-get-started.md [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+
+You can find links to various notebooks and videos describing analyses done on Testgrid data [here](content.md#testgrid).
+
+### Analyzing Prow
+
+[Prow](https://prow.k8s.io/) logs represent a rich source of information for automated triaging and root cause analysis. But unfortunately, these logs are noisy data types, i.e. two logs of the same kind but from two different sources may be different enough at a character level that traditional comparison methods are insufficient to capture this similarity. To overcome this issue, we use the Prow logs to learn log templates that denoise log data and help improve performance on downstream ML tasks.
```
Comment (MichaelClifford) [inline docs/kubernetes-contributor-get-started.md:17]: I don't think we currently "use the Prow logs to learn log templates that denoise log data and help improve performance on downstream ML tasks"

**Review by MichaelClifford — CHANGES_REQUESTED** [review]

File: docs/data-science-get-started.md [hunk 8]
```diff
@@ -0,0 +1,37 @@
+# Engage as a Data Scientist
```
[the remark this answers is not in the document]
Comment (oindrillac) [reply docs/data-science-get-started.md:1]: We are not necessarily giving them documentation on how to Contribute here, only docs on how to get started with the project. Maybe we can say "Get Involved as a ....." ?

[the remark this answers is not in the document]
Comment (MichaelClifford) [reply docs/data-science-get-started.md:1]: Yep, I like "get involved"

File: docs/kubernetes-contributor-get-started.md (hunk 7 above)
Comment (aakankshaduggal) [reply docs/kubernetes-contributor-get-started.md:17]: Makes sense, fixed that :+1:

**Review by isabelizimm — COMMENTED** [review]
/lgtm

**Comment by aakankshaduggal** [comment]
Thank you @MichaelClifford and @isabelizimm for your feedback. We have implemented the changes.

**Comment by oindrillac** [comment]
bump @MichaelClifford, is this good to 🚢

**Pull request merged**